src/utils/internal/getCallFrame.test.ts

```diff
@@ -0,0 +1,163 @@
+import { getCallFrame } from './getCallFrame'
+
+const mockStack = jest.fn()
+jest.spyOn(window, 'Error').mockImplementation(() => ({
+  name: 'mockError',
+  message: '',
+  stack: mockStack(),
+}))
+
+test('Node on Linux and macOS error stack', () => {
+  // version 1
+  mockStack.mockImplementationOnce(() =>
+    [
+      'Error: ',
+      '    at getCallFrame (/Users/mock/github/msw/lib/umd/index.js:3735:22)',
+      '    at Object.get (/Users/mock/github/msw/lib/umd/index.js:3776:29)',
+      '    at Object.<anonymous> (/Users/mock/github/msw/test/msw-api/setup-server/printHandlers.test.ts:13:8)', // <-- this one
+      '    at Runtime._execModule (/Users/mock/github/msw/node_modules/jest-runtime/build/index.js:1299:24)',
+      '    at Runtime._loadModule (/Users/mock/github/msw/node_modules/jest-runtime/build/index.js:898:12)',
+      '    at Runtime.requireModule (/Users/mock/github/msw/node_modules/jest-runtime/build/index.js:746:10)',
+      '    at jasmine2 (/Users/mock/github/msw/node_modules/jest-jasmine2/build/index.js:230:13)',
+      '    at runTestInternal (/Users/mock/github/msw/node_modules/jest-runner/build/runTest.js:380:22)',
+      '    at runTest (/Users/mock/github/msw/node_modules/jest-runner/build/runTest.js:472:34)',
+    ].join('\n'),
```

Comment:
Alternatively, we could return a template string literal that preserves line breaks. Do you think that would be different from the current approach?

Reply:
I added `console.warn(frames)` to the function and copy&pasted the arrays from console :) Otherwise the only difference would be in indentation.

Reply:
oh, and I couldn't do the `// <-- this one` comment .. I used it only to guide me to make the assertion, not sure how useful it is to keep those comments for future debugging 🤷‍♂️

=> not changing now, but if you tell me you really really prefer template string literal without indentation and without line comments, I can convert.